### PR TITLE
devsign lab

### PR DESCRIPTION
**Pull request by deetolman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "10_reduxPerf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest --verbose --passWithNoTests",
```
Comment (grayson073) [inline package.json]:
```suggestion
    "test": "jest --verbose --passWithNoTests",
    "pretest": "npm run lint",
    "lint": "eslint .",
```

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+AUTH0_CLIENT_ID=GtAdoqPmwaffc8XYaVFP3u2WkeSvuUOh
```
Comment (MariahAdams) [inline .env]: Remove env from Github, add to gitignore and include ```.env.example``` file for the next dev.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "10_reduxPerf",
```
Comment (MariahAdams) [inline package.json]: Update name

File: package.json (hunk 1 above)
Comment (MariahAdams) [reply package.json]: Remove --passWithNoTests flag.

**Pull request merged**